oaknational/Oak-Web-Application
# [PUPIL-816] Add min widths to prevent cumulative layout shift on year selection page

## Description

Music year: 1993

Added an extra oak box around the year links that has a min-width slightly larger than the rendered text size. 

Too be honest, I'm not 100% certain this is going to fix the web vitals score, I've used the performance tab to look for layout shifts and I still see one when the font loads, although it's not the whole white box now, it's just the "Welcome to Oak!" div.

I think we merge this and see if the score has improved in a week, if not we can look at this in more detail and maybe draft in some more opinions.

## How to test

1. Go to https://deploy-preview-2546--oak-web-application.netlify.thenational.academy/pupils/years
2. Refresh the page using Cmd (Command) + Shift + R
3. You should see the page reload without the white box changing size

| Current | New |
|----------|----------|
| ![cls years before](https://github.com/oaknational/Oak-Web-Application/assets/587241/5fbd326d-c6f2-440f-b677-fdd89c4672eb) | ![cls years after](https://github.com/oaknational/Oak-Web-Application/assets/587241/2e506c70-c5ec-4b6a-b842-c06d82f425e3) |



## Checklist
